### DISCUSSION TITLE
After Upgrading WebDriver IO  from V7 to V8 we are getting the certificate error .

### DISCUSSION THREAD

**Discussion by givemesomecode** [webdriverio/webdriverio#10969] [Q&A]
This occur frequently but not consistent, are there any configs to do. we are using V8 with new Chrome for Test and cucmber framework with Type Script

Error: Error: unable to get local issuer certificate
    at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34)
    at TLSSocket.emit (node:events:513:28)
    at TLSSocket.emit (node:domain:489:12)
    at TLSSocket._finishInit (node:_tls_wrap:959:8)
Error: unable to get local issuer certificate
    at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34)
    at TLSSocket.emit (node:events:513:28)
    at TLSSocket.emit (node:domain:489:12)
    at TLSWrap.ssl.onhandshakedone (node:_tls_wrap:743:12) {
  code: 'UNABLE_TO_GET_ISSUER_CERT_LOCALLY'
}

**Comment by christian-bromann** [comment]
Are you behind a corporate firewall? If so can you try setting `HTTP_PROXY` or `HTTPS_PROXY`?

**Comment by nborton4** [comment]
You can force Node.js to skip the verification. Just execute export NODE_TLS_REJECT_UNAUTHORIZED=0

**Reply by ChetanPratap0** [reply]
![image](https://github.com/webdriverio/webdriverio/assets/108003815/55485227-f662-40be-af93-856e4a3f669b)
I encounters same issue while running my scripts on chrome browser. I tried disabling certificated verification but still facing the same issue. Could this be due to difference in the chrome driver downloaded by webdriverIO and the chrome version on my local machine

**Comment by ShridharaRavi** [comment]
@christian-bromann  - Is this issue resolved ? If yes could you please suggest how ? having same issue in my work environment

**Reply by christian-bromann** [reply]
This issue is not resolved, see https://github.com/webdriverio/webdriverio/issues/11250

**Reply by Sugabeca** [reply]
Good night Christian! @christian-bromann 

I'm getting this same issue I'm using the most recent webdriverio so the files are not matching in order to change them and apply the fix from your last comment #11250 (https://github.com/webdriverio/webdriverio/commit/ee45c83a8abe33972af8fabe183db130316379c5) 
could you please add an update of the files using the latest version?

Thanks!
